lib/manager/sbt/readme.md

```diff
@@ -0,0 +1,240 @@
+## Overview
+
+#### Name of package manager
+
+sbt
+
+---
+
+#### What language does this support?
+
+Scala
+
+---
+
+#### Does that language have other (competing?) package managers?
+
+Maven
+
+## Package File Detection
+
+#### What type of package files and names does it use?
+
+As far as I know, there is 2 package files:
+- One for main dependencies and some configurations (https://www.scala-sbt.org/1.0/docs/Basic-Def.html)
+- One for plugin dependencies and plugin configurations (https://www.scala-sbt.org/1.0/docs/Plugins.html)
+
+---
+
+#### What [fileMatch](https://renovatebot.com/docs/configuration-options/#filematch) pattern(s) should be used?
+
+`['build\\.sbt', 'plugins\\.sbt']`
+
+Note that `plugins.sbt` file is usually in a `project/` subdirectory.
+
+---
+
+#### Is it likely that many users would need to extend this pattern for custom file names?
+
+Not likely.
+
+---
+
+#### Is the fileMatch pattern likely to get many "false hits" for files that have nothing to do with package management?
+
+Not likely. The extension `.sbt` should not be used for other files.
+
+## Parsing and Extraction
+
+#### Can package files have "local" links to each other that need to be resolved?
+
+No
+
+#### Is there reason why package files need to be parsed together (in serial) instead of independently?
+
+No
+
+---
+
+#### What format/syntax is the package file in? e.g. JSON, TOML, custom?
+
+Scala syntax
+
+---
+
+#### How do you suggest parsing the file? Using an off-the-shelf parser, using regex, or can it be custom-parsed line by line?
+
+I think reading the files with Scala would be the easier. Due to the fact that the files are plain Scala code, the
```

Comment:
https://github.com/scalameta/scalameta running on http://www.scala-js.org could be a way to parse Scala without depending on a JDK.
E.G. https://astexplorer.net/#/gist/027dce11e6927b4ad39ea097ce6289b9/ea4d1048f32063a71d727d01178ae6d01087a62f